src/main/java/core/basesyntax/services/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void writeToFile(String filePath, String report) {
+        try {
+            Files.writeString(Paths.get(filePath), report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find file by path" + filePath);
```

Comment:
Consider using a custom exception, such as `InvalidDataException`, instead of a generic `RuntimeException` for better error specificity and handling when an IO error occurs.